notebooks/05-operations/01-druid-logs-configuration.ipynb

```diff
@@ -0,0 +1,524 @@
+    "\n",
+    "Launch this tutorial and all prerequisites using the `jupyter` profile of the Docker Compose file for Jupyter-based Druid tutorials. For more information, see the Learn Druid repository [readme](https://github.com/implydata/learn-druid).\n",
+    "\n",
+    "__DO NOT__ use the `jupyter-druid` profile with this tutorial as it will conflict with your locally running copy."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "151ede96-348f-423c-adc7-0e23c6910b9e",
+   "metadata": {},
+   "source": [
+    "## Initialization\n",
+    "\n",
+    "To use this notebook, you will need to run Druid locally.\n",
+    "\n",
+    "You will also make extensive use of the terminal, which you can place alongside this notebook or on another screen.\n",
+    "\n",
+    "### Install tools\n",
+    "\n",
+    "Open a local terminal window.\n",
+    "\n",
+    "If you have not already install `wget` or `multitail`, run the following commands to install these tools using `brew`.\n",
+    "\n",
+    "```bash\n",
+    "brew install multitail ; brew install wget\n",
+    "```\n",
+    "\n",
+    "Run the following command to pull the default configuration for `multitail` to your home folder. Do not run this command if you are already running `multitail` as it will overwrite your own configuration.\n",
+    "\n",
+    "```bash\n",
+    "curl https://raw.githubusercontent.com/halturin/multitail/master/multitail.conf > ~/.multitailrc\n",
+    "```\n",
+    "\n",
+    "### Install Apache Druid\n",
+    "\n",
+    "Run the following to create a dedicated folder for learn-druid in your home directory:\n",
+    "\n",
+    "```bash\n",
+    "cd ~ ; mkdir learn-druid-local\n",
+    "cd learn-druid-local\n",
+    "```\n",
+    "\n",
+    "Now pull and decompress a compatible version of Druid.\n",
+    "\n",
+    "```bash\n",
+    "wget https://dlcdn.apache.org/druid/29.0.0/apache-druid-29.0.0-bin.tar.gz\n",
+    "tar -xzf apache-druid-29.0.0-bin.tar.gz\n",
+    "```\n",
+    "\n",
+    "Use the following commands to rename the folder.\n",
+    "\n",
+    "```bash\n",
+    "mv apache-druid-29.0.0 apache-druid\n",
+    "cd apache-druid\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1d07feb7-92dd-42a9-ad6c-c89d5fdad8d5",
+   "metadata": {},
+   "source": [
+    "# Review the log file configuration\n",
+    "\n",
+    "The log file configuration is set in the `log4j2.xml` alongside Druid's own configuration files.\n",
+    "\n",
+    "Remembering that Druid has multiple configuration file locations out-of-the-box, run this command to view the `auto` configuration file for logs that's used by the `learn-druid` script:\n",
+    "\n",
+    "```bash\n",
+    "more ~/learn-druid-local/apache-druid/conf/druid/auto/_common/log4j2.xml\n",
+    "```\n",
+    "\n",
+    "Within the [`Configuration`](https://logging.apache.org/log4j/log4j-2.4/manual/configuration.html#ConfigurationSyntax) section you will see:\n",
```

Comment:
This section appears to be out of place. Is it referring to the configuration file syntax or the official documentation for  log4j?